registry/storage/cache/memory/memory_test.go

```diff
@@ -3,11 +3,11 @@ package memory
 import (
 	"testing"
 
-	"github.com/docker/distribution/registry/storage/cache"
+	"github.com/docker/distribution/registry/storage/cache/testutils"
```

Comment:
If I name it `check`, then the entry point `CheckBlobDescriptorCache` will stutter and golint will complain.

Normally in a case like this, we'd rename `CheckBlobDescriptorCache` to `BlobDescriptorCache`, but that doesn't feel like the right name for a function that carries out checks. What do you think?